services/update.py

```diff
@@ -179,30 +178,7 @@ def get_update(self):
                 -- add-on, and the ORDER BY and LIMIT 1 clauses below make it
                 -- unlikely that we'll be examining a large number of rows,
                 -- so this is fairly cheap.
```

Comment:
That comment should be updated since you're removing the `WHEN` clauses.

Reply:
Oh, right, I missed that this comment was related to the `CASE`.